docs/getting_started.md

```diff
@@ -49,65 +46,69 @@ If you are not interested in backing up any data, the `--gcs.bucket` flag can si
 * _[Example Kubernetes manifest with GCS upload](../kube/manifests/prometheus-gcs.yaml)_
 * _[Details & Config for other object stores](./storage.md)_
 
-### Query Access
+### [Store API](components/store.md)
+
+The Sidecar comes with a [gRPC](https://grpc.io/) API called the _[Store API](components/store.md)_. The Store API allows you to query metric data in Prometheus, and data backed up into the Object Store bucket.
 
-Thanos comes with a highly efficient gRPC-based Store API for metric data access across all its components. The sidecar implements it in front of its connected Prometheus server. While it is ready to use with the above example, we must additionally configure the sidecar to join a Thanos cluster.
+In order to expose the Store API, we also need to put the Sidecar into '_cluster_' mode. A cluster can be made up of one or more Sidecars. Clustering uses the [Gossip Protocol](https://en.wikipedia.org/wiki/Gossip_protocol) to allow Thanos components- including the Sidecar itself- to be able to find other Sidecars. Clustering is explained in greater detail in the [Communication Between Components](#communication-between-components) section.
```

Comment:
This section is entirely false. There is literally NO clustering needed for sidecars, and we have to fix this confusion ASAP. In fact we will get rid of gossip as this only causes problems: https://github.com/improbable-eng/thanos/issues/484

Reply:
Whole Communication between components needs to be revisited as well ):

Reply:
TL;DR only querier needs to know where storeAPIs are (sidecars, rulers, store gateways or other qieriers)

Reply:
Ah! Thanks for the clarification. I was going to get to the Component comms later today but it looks like I need to revisit the entire section here. 🤔